docs/source/install/monitor_without_docker.rst

```diff
@@ -315,8 +321,10 @@ Depends if you installed Grafana from a repository (yum install), or if you down
 different in the rest of the steps.
 
 2. Access Scylla-Grafana-monitoring directory
+.. code-block:: shell
+   :substitutions:
```

Comment:
```suggestion

.. code-block:: shell
   :substitutions:
```

Reply:
@amnonh Fix for the warning error ``scylla-monitoring/docs/source/install/monitor_without_docker.rst:325: WARNING: Unexpected indentation.`` that breaks the build.